modules/modelLoader/stableDiffusion3/StableDiffusion3EmbeddingLoader.py

```diff
@@ -1,61 +1,62 @@
 import os
 import traceback
 
-import torch
+from modules.model.StableDiffusion3Model import StableDiffusion3Model
+from modules.util.ModelNames import EmbeddingName, ModelNames
+
 from safetensors.torch import load_file
-from torch import Tensor
 
-from modules.model.StableDiffusion3Model import StableDiffusion3Model
-from modules.util.ModelNames import ModelNames, EmbeddingName
+import torch
+from torch import Tensor
 
 
 class StableDiffusion3EmbeddingLoader:
     def __init__(self):
-        super(StableDiffusion3EmbeddingLoader, self).__init__()
+        super().__init__()
 
     def __load_embedding(
-            self,
-            embedding_name: str,
+        self,
+        embedding_name: str,
     ) -> tuple[Tensor, Tensor, Tensor] | None:
         if embedding_name == "":
             return None
 
         try:
```

Comment:
In other places you replaced these try blocks with an empty except block with `contextlib.suppress` Why is this not done everywhere?

Reply:
I'll search for other places -- the linter did it automatically in those places.

Reply:
Done!